dataland-frontend/tests/e2e/utils/GeneralUtils.ts

```diff
@@ -41,7 +41,7 @@ export function compareObjectKeysAndValuesDeep(
   for (const key of keysA) {
     const newPath = path ? `${path}.${key}` : key;
 
-    if (!keysB.includes(key)) {
+    if (!keysB.includes(key) && !key.endsWith('publicationDate')) {
```

Comment:
This if breaks the expected behavior of the function judging by name and comment.

Reply:
ist jetzt konfigurierbar